functions/public/scripts/profile.js

```diff
@@ -92,7 +92,7 @@ Profile.prototype.saveProfile = function (year, course, university) {
         timeout: 2000
       };
       //self.signInSnackbar.MaterialSnackbar.showSnackbar(data);
-      window.alert("profile saved!");
+      window.history.back();
```

Comment:
This only takes the user to the page they were on, before they came to the profile page. What if the user was on the questions page, then type `pastpaperhub.com/profile` in the address bar, then changes their profile and clicked `save`, I think they would be shown int questions page instead of the home page. A better way to handle this is by using something like `window.location.href = "/"`

Reply:
Will work on this in the next PR